tests/unit_tests/pyanaconda_tests/core/test_util.py

```diff
@@ -815,6 +817,134 @@ def test_dracut_eject(self):
                     file_contents = "\n".join(f.readlines())
                     assert "eject " + devname in file_contents
 
+    @patch("pyanaconda.core.util.open")
+    def test_is_stage2_on_nfs(self, mock_open):
+        """Test check for installation running on nfs."""
+        nfs_source_mounts = """
```

Comment:
It's huge text strings. I will not insist, but I would suggest to put the mounts in a separate file, which we can include here.

Reply:
It is, I wanted to have real output there but I think I'll rather trim them to a few lines?

Reply:
Picked only a few lines.